cvmfsInstall.sh

```diff
@@ -188,6 +188,8 @@ rm -f $BASEDIR/latest
 ln -s $(grep "^nweek-" ${CMS_BOT_DIR}/ib-weeks | tail -1) $BASEDIR/latest
 
 if $CVMFS_INSTALL ; then
+  # Remove RPMs before publishing
+  rm -rf $WORKDIR/*/var/cmspkg/rpms || true
```

Comment:
@iarspider , any idea why https://github.com/cms-sw/cms-bot/blob/master/cvmfsInstall.sh#L150 is not enough?

Reply:
For some jobs (the ones triggered by cmsrep webhook) we do not pass `RELEASE_FORMAT` (which is assigned to `RELEASE_NAME`), thus the condition at [line 144](https://github.com/cms-sw/cms-bot/blob/master/cvmfsInstall.sh#L144) evaluates to false. I guess I can move cleanup at line 150 below [line 160](https://github.com/cms-sw/cms-bot/blob/master/cvmfsInstall.sh#L160) so that it is always executed...